StripePayments/StripePayments/Source/API Bindings/STPAPIClient+Payments.swift

```diff
@@ -1145,7 +1145,8 @@ extension STPAPIClient {
                         shared_lastError = error
                     }
                     if let paymentMethods = deserializer?.paymentMethods {
-                        shared_allPaymentMethods.append(contentsOf: paymentMethods)
+                        // For unknown reasons, `append(contentsOf:` here sometimes causes an EXC_BAD_INSTRUCTION if you repeatedly run tests
+                        paymentMethods.forEach { shared_allPaymentMethods.append($0) }
```

Comment:
since these are both of type `[STPPaymentMethod]` and `shared_allPaymentMethods` is a var, maybe we can avoid the `forEach` by using:
```
shared_allPaymentMethods += paymentMethods
```

Reply:
Maybe, I didn't try that. But I don't want to re-run the tests 20 times to find out if this is flakey or not >_>